src/main/java/io/github/mountainrange/mule/gameplay/Grid.java

```diff
@@ -207,4 +209,35 @@ public void removePlayer() {
 	public Point2D getPlayerPosition() {
 		return playerPosition;
 	}
+
+	@Override
```

Comment:
Yes, it should only iterate over tiles. I needed a convenient way to give players points for each tile they owned (hence implementing `Iterable<Tile>`).